MEEW-Client/MEEW-Client/Networks/APIServices/CharacterListService.swift

```diff
@@ -10,15 +10,10 @@ import Alamofire
 
 struct CharacterListService {
```

Comment:
1. 싱글톤 객체의 사용은 class가 적합하므로 변경하면 좋을 것 같습니다!
```suggestion
class CharacterListService {
```

2. 추가로 인스턴스 재생성을 막기 위해 initializer를 private하게 제한해줬으면 좋겠어요
```swift
private init() {}
```

Reply:
포스트맨 안찍혀잉....